packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -0,0 +1,371 @@
+ *
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import "@ui5/webcomponents/dist/ViewSettingsDialog";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.fiori.ViewSettingsDialog
+ * @extends UI5Element
+ * @tagname ui5-view-settings-dialog
+ * @since 1.0.0-rc.16
+ * @public
+ */
+class ViewSettingsDialog extends UI5Element {
+	constructor() {
+		super();
+		this.i18nBundle = getI18nBundle("@ui5/webcomponents-fiori");
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get metadata() {
+		return metadata;
+	}
+
+	static get dependencies() {
+		return [
+			Bar,
+			Button,
+			Dialog,
+			List,
+			StandardListItem,
+			GroupHeaderListItem,
+		];
+	}
+
+	static get template() {
+		return ViewSettingsDialogTemplate;
+	}
+
+	static get styles() {
+		return viewSettingsDialogCSS;
+	}
+
+	static async onDefine() {
+		await fetchI18nBundle("@ui5/webcomponents-fiori");
+	}
+
+	get _dialogTitle() {
+		return this.i18nBundle.getText(VSD_DIALOG_TITLE_SORT);
+	}
+
+	get _okButtonLabel() {
+		return this.i18nBundle.getText(VSD_SUBMIT_BUTTON);
+	}
+
+	get _cancelButtonLabel() {
+		return this.i18nBundle.getText(VSD_CANCEL_BUTTON);
+	}
+
+	get _resetButtonLabel() {
+		return this.i18nBundle.getText(VSD_RESET_BUTTON);
+	}
+
+	get _ascendingLabel() {
+		return this.i18nBundle.getText(VSD_ORDER_ASCENDING);
+	}
+
+	get _descendingLabel() {
+		return this.i18nBundle.getText(VSD_ORDER_DESCENDING);
+	}
+
+	get _sortOrderLabel() {
+		return this.i18nBundle.getText(VSD_SORT_ORDER);
+	}
+
+	get _sortByLabel() {
+		return this.i18nBundle.getText(VSD_SORT_BY);
+	}
+
+	get _isPhone() {
+		return isPhone();
+	}
+
+	get _sortAscending() {
+		return !this.sortDescending;
+	}
+
+	/**
+	 * Determines disabled state of the <code>Reset</code> button.
+	 */
+	get _disableResetButton() {
+		return this._dialog && JSON.stringify(this._currentSettings) === JSON.stringify(this._initialSettings);
+	}
+
+	/**
+	 * Returns the current settings (current state of all lists).
+	 */
+	get _settings() {
+		const settings = {},
+			  sortOrderSelected = this._sortOrder.getSelectedItems(),
+			  sortBySelected = this._sortBy.getSelectedItems();
+
+		settings.sortOrder = sortOrderSelected.length ? sortOrderSelected[0] : undefined;
+		settings.sortBy = sortBySelected.length ? sortBySelected[0] : undefined;
+		return settings;
+	}
+
+	/**
+	 * Opens the dialog. On first call does initialization of the control.
+	 */
+	open() {
+		if (!this._dialog) {
+			this._sortOrder = this.shadowRoot.querySelector("[ui5-list][sort-order]");
+			this._sortBy = this.shadowRoot.querySelector("[ui5-list][sort-by]");
+			this._initialSettings = this._settings;
+			this._currentSettings = this._initialSettings;
+			this._confirmedSettings = this._initialSettings;
+			this._dialog = this.shadowRoot.querySelector("[ui5-dialog]");
+		} else {
+			this._restoreSettings(this._confirmedSettings);
+		}
+		this._dialog.open();
```

Comment:
This is not working anymore because of the changes in the dialog/popover.
It should be:
```this._dialog.show();```

Reply:
Done